Appraisals

```diff
@@ -0,0 +1,4 @@
+appraise "rails-5" do
```

Comment:
I added a gem called [Appraisal](https://github.com/thoughtbot/appraisal) by thoughtbot that helps you test against multiple configurations of your project 🎉 

This allows us to set up our travis.yml to test across all combinations of ruby 2.7/2.6 and Rails 5/6